lib/tree.js

```diff
@@ -236,15 +236,18 @@ class Tree {
 
 	/**
 	 * Exclude modules from tree using RegExp.
-	 * @param  {Object} tree
-	 * @param  {Array} excludeRegExp
+	 * @param {Object} tree
+	 * @param {(Array|false)} includeRegExp
+	 * @param {(Array|false)} excludeRegExp
 	 * @return {Object}
 	 */
-	exclude(tree, excludeRegExp) {
-		const regExpList = excludeRegExp.map((re) => new RegExp(re));
+	exclude(tree, includeRegExp, excludeRegExp) {
+		const includeList = includeRegExp ? includeRegExp.map((re) => new RegExp(re)) : false;
+		const excludeList = excludeRegExp ? excludeRegExp.map((re) => new RegExp(re)) : false;
 
 		function regExpFilter(id) {
-			return regExpList.findIndex((regexp) => regexp.test(id)) < 0;
+			return (!includeList || includeList.findIndex((re) => re.test(id)) >= 0) &&
```

Comment:
This logic means, only filenames declared in `--include` are included, other files, even those not `--exclude`-d, won't be checked?